### PR TITLE
Upload to CDN

### PR DESCRIPTION
**Pull request by himadrisingh**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -116,6 +114,18 @@ jobs:
...
+        - echo "version=${TRAVIS_TAG}" > metadata.txt
+        - echo "build_url=${TRAVIS_BUILD_WEB_URL} >> metadata.txt
+        - echo "sha=$(git rev-parse) >> metadata.txt
+
+        - echo ${GCP_TOKEN} > gcp.json
+        - gcloud auth activate-service-account --key-file gcp.json
+        - gcloud config set project rilldata
+        - gsutil cp rilldata/rill-macos-x64 gs://prod-cdn.rilldata.com/rill/latest/macos/rill
+        - gsutil cp rilldata/rill-linux-x64 gs://prod-cdn.rilldata.com/rill/latest/linux/rill
+        - gsutil cp rilldata/rill-win-x64.exe gs://prod-cdn.rilldata.com/rill/latest/win/rill
```
Comment (himadrisingh) [inline .travis.yml:126]:
```suggestion
        - gsutil cp rilldata/rill-macos-x64 gs://prod-cdn.rilldata.com/rill/latest/macos-x64/rill
        - gsutil cp rilldata/rill-linux-x64 gs://prod-cdn.rilldata.com/rill/latest/linux-x64/rill
        - gsutil cp rilldata/rill-win-x64.exe gs://prod-cdn.rilldata.com/rill/latest/win-x64/rill
```

File: .travis.yml [hunk 2]
```diff
@@ -116,6 +114,18 @@ jobs:
         - echo "Building vercel/pkg binary"
         - node build-tools/replace_package_type.cjs module commonjs && npm run build-vercel-pkg
 
+        - echo "version=${TRAVIS_TAG}" > metadata.txt
```
Comment (AdityaHegde) [inline .travis.yml:117]: Moving this to a script might be better. This file already growing too big.

**Review by AdityaHegde — APPROVED** [review]
Apart from moving things to a script this looks good.

File: .travis.yml (hunk 2 above)
Comment (himadrisingh) [reply .travis.yml:117]: do we have any metadata.txt already

**Pull request merged**